### PR TITLE
Week 2 - Project-news-site -Jessi Nygren Walhed 

### PR DESCRIPTION
**Pull request by hemmahosjessi**
Week 2 ⭐️ 
https://reverent-shockley-cab6a2.netlify.app/

Most proud of the fact that I made it! Had a lot of struggle with Flexbox. Think I experience Grid easier. But needs more exploration.

### CODE REVIEW

**Review by silvertejp89 — APPROVED** [review]
The webbpage look very complete, all details seems to be thought out and properly styled. Looks proffessional, I´m impressed! 
There could be more describing comments inte the code to really serve this on a silver platter for the reader, but overall I think I can read both yout HTML and CSS pretty clearly! 🥇

File: code/foxlogo.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+            <g id="Footer-2" transform="translate(0.000000, 1572.000000)">
+                <g id="Group" transform="translate(530.000000, 58.000000)">
+                    <text id="🦊">
+                        <tspan x="8" y="40">🦊</tspan>
+                    </text>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
[the remark this answers is not in the document]
Comment (hemmahosjessi) [reply code/foxlogo.svg:15]: Hm not sure if I understand... That is probably somethings that adds when you publish... Svg things. And apple emoji stuff code things. Haha. 

Hm the thing with the empty line? I dont follow Could you show me?

File: code/index.html [hunk 2]
```diff
@@ -4,19 +4,116 @@
...
-    <section class="">
-      <!-- The other news  -->
+    <section class="small-news-container">
+      <div class="small-news-item"> 
+        <a href="https://www.wired.com/story/gadget-lab-podcast-506/"> 
+          <img class="small-news-img" src="./color-img.png" alt="#"/>
+        <h3>Humanity’s Obsession With Color Has Shaped Our Modern World.</h3>
+        </a>
+      </div>
+      <div class="small-news-item">
```
[the remark this answers is not in the document]
Comment (hemmahosjessi) [reply code/index.html:61]: Had to look that kebab-case style up... Is the kebab when you write it all in small caps? Or did I get that wrong?

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,491 @@
...
+    margin-right: 2rem;
+  }
+
+  .footer a:hover {
+    position: relative;
+    transform: scale(1.1);
+    transition-property: all;
+    transition-duration: 0.4s;
+  }
+}
```
[the remark this answers is not in the document]
Comment (hemmahosjessi) [reply code/style.css:491]: Hm didn´t understand that endline thing...

**Pull request closed without merging**